src/utils/getAccessToken.ts

```diff
@@ -0,0 +1,11 @@
+"use server";
```

Comment:
accessToken now available in Session Object

Reply:
Done